src/main/java/Duke.java

```diff
@@ -1,59 +1,58 @@
 import java.util.Scanner;
 
 public class Duke {
-
-    private final static String DONE = "DONE";
-
     private final static int EXIT = 0;
```

Comment:
Great use of const to remove ambiguity from magic numbers!

Reply:
Thanks!